### PR TITLE
chore: fix release notification step

### PR DESCRIPTION
**Pull request by alexanderniebuhr**
## Changes

- this should fix the notification script

## Testing

- existing

## Docs

- not needed

### CODE REVIEW

File: scripts/notify/index.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ function singularlize(text) {
 const packageMap = new Map();
 async function generatePackageMap() {
 	const packageRoot = new URL('../../packages/', import.meta.url);
-	const packages = await glob(['*/package.json', '*/*/package.json'], {
+	const packages = await glob('*/package.json', {
```
Comment (alexanderniebuhr) [inline scripts/notify/index.js:85]: Looking at the current structure from the repo, this glob should be enough, I don't think we have two level nested packages 🤔

**Review by delucis — APPROVED** [review]
Seems plausible! 👍

File: scripts/notify/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { readFile } from 'node:fs/promises';
 import { fileURLToPath } from 'node:url';
-import { globby as glob } from 'globby';
+import glob from 'tiny-glob';
```
Comment (ematipico) [inline scripts/notify/index.js:3]: I would suggest using https://www.npmjs.com/package/tinyglobby, which is even smaller

**Review by bluwy — APPROVED** [review]
LGTM. I think we can switch to tinyglobby altogether later (since we're using a mix of tiny-glob and fast-glob currently)

**Comment by alexanderniebuhr** [comment]
I probably should have made this PR against `main` instead of `next` 🤔

**Comment by bluwy** [comment]
I think you can change it from the edit UI directly if you'd like

File: scripts/notify/index.js (hunk 2 above)
Comment (alexanderniebuhr) [reply scripts/notify/index.js:3]: As said in [this comment](https://github.com/withastro/adapters/pull/395#pullrequestreview-2309960412) let's switch altogether in a follow-up :)

**Pull request merged**